doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
+the python script defined in section: The Application Script.
+To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
+The function returns a ``RunSettings`` object.
+A ``RunSettings`` allows you to configure
+the run settings of a SmartSim entity.
+We initialize a RunSettings object and
+specify the path to the application file,
+`application_script.py`, to the argument
+``exe_args``, and the run command to ``exe``.
 
 .. note::
+  Change the `exe_args` argument to the path of the application script
+  on your file system to run the example.
 
-  Pinning _only_ affects the co-located deployment because both the application and the database
-  are sharing the same compute node. For the clustered deployment, a shard occupies the entirety
-  of the node.
+Use the ``RunSettings`` helper functions to
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the Model to run on a single compute node.
 
-Redis
-=====
+.. code-block:: python
 
-.. _Redis: https://github.com/redis/redis
-.. _RedisAI: https://github.com/RedisAI/RedisAI
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/clustered_model.py")
+    # Configure RunSettings object
+    model_settings.set_nodes(1)
 
-The ``Orchestrator`` is built on `Redis`_. Largely, the job of the ``Orchestrator`` is to
-create a Python reference to a Redis deployment so that users can launch, monitor
-and stop a Redis deployment on workstations and HPC systems.
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+.. code-block:: python
 
-Redis was chosen for the Orchestrator because it resides in-memory, can be distributed on-node
-as well as across nodes, and provides low latency data access to many clients in parallel. The
-Redis ecosystem was a primary driver as the Redis module system provides APIs for languages,
-libraries, and techniques used in Data Science. In particular, the ``Orchestrator``
-relies on `RedisAI`_ to provide access to Machine Learning runtimes.
+    # Initialize a SmartSim Model
+    model = exp.create_model("colo_model", model_settings)
 
-At its core, Redis is a key-value store. This means that put/get semantics are used to send
-messages to and from the database. SmartRedis clients use a specific hashing algorithm, CRC16, to ensure
-that data is evenly distributed amongst all database nodes. Notably, a user is not required to
-know where (which database node) data or Datasets (see Dataset API) are stored as the
-SmartRedis clients will infer their location for the user.
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_tcp()`` function.
```

Comment:
_tcp does not do uds.  _uds does.  We should change language or change example.

Reply:
oop